src/functions/operators.js

```diff
@@ -0,0 +1,98 @@
+// @flow
+import defineFunction, {ordargument} from "../defineFunction";
+import buildCommon from "../buildCommon";
+import mathMLTree from "../mathMLTree";
+
+import * as html from "../buildHTML";
+import * as mml from "../buildMathML";
+
+// \operatorname
+// amsopn.dtx: \mathop{#1\kern\z@\operator@font#3}\newmcodes@
+defineFunction({
```

Comment:
Thanks for defining these new functions using the new `defineFunction`.  Let me know if you have any ideas of how to make this new structure better.